packages/victory-errorbar/src/helper-methods.js

```diff
@@ -198,7 +211,8 @@ const getBaseProps = (props, fallbackProps) => {
       scale,
       style: style.data,
```

Comment:
Missing a check for `disableInlineStyles` with the `style` prop here.

Reply:
I'm not sure some of these checks are doing anything. We can add another just to be safe, but as you noted below, any of the components that are using `Helpers.evaluateStyle` already have this conditional in place.

Reply:
I'll add it in for consistency - I'm not sure these conditionals are doing much in most cases when we already have the check in the `evaluateStyle` helper.